components/brave_wallet/browser/ethereum_provider_impl.h

```diff
@@ -257,6 +258,7 @@ class EthereumProviderImpl final
                            const absl::optional<std::string>& domain_hash,
                            const absl::optional<std::string>& primary_hash,
                            bool is_eip712,
+                           bool is_eth_sign,
```

Comment:
Please do not add `is_eth_sign` when `is_eip712` is false already means doing `eth_sign`

Reply:
see below